mico-core/src/main/java/io/github/ust/mico/core/persistence/MicoApplicationRepository.java

```diff
@@ -41,5 +42,14 @@
 
     @Depth(3)
     Optional<MicoApplication> findByShortNameAndVersion(String shortName, String version);
+
+    /**
+     * Find all applications that are using the same service
```

Comment:
JavaDoc `@param`should be used like this `* @param shortName the shortName of the {@link MicoService}.`